### DISCUSSION TITLE
Does ZLS works with Code::Blocks IDE? Can anyone develop a Code::Blocks Plugin?

### DISCUSSION THREAD

**Discussion by LMS-PT** [zigtools/zls#1593] [Ideas]
I am using the the Code::Blocks IDE to learn a little bit of C/C++ on Windows.  (Just basic stuff)
I like it because it is OpenSource and they have an an installer that additionally includes the GCC/G++/GFortran compilers through MinGW-W64... I find it a very simple/convenient way of "entering this world"...

Since Zig compiler is targeting the C/C++ space and also compiles C/C++, I think it could be a good idea if it would be easally used from the Code::Blocks IDE.
So my questions are:
  - Are there Zig plugins for Code::Blocks?
  - If not, Anyone interested in developing it?

Note: I am not a software developer. I am a mechanical engineer who works on product design.
(Complex products quite often need to include some sensors/Microcontrollers/Single-Board Computers, ...)
(I am stuck to Windows because of the 3D CAD tools I need to use for my work.)

**Comment by llogick** [comment]
None that I'm aware of. The best place to ask would be in one of the [communities](https://github.com/ziglang/zig/wiki/Community) (The Discord server even has a "requests" channel/section).
